### ISSUE TITLE
Responses for events.ics should have a CORS header appended

### ISSUE THREAD

**Issue by blandin** [GetTogetherComm/GetTogether#186]
For JavaScript on other sites to access and parse data in events.ics, an `Access-Control-Allow-Origin` header needs to be appended to the response. This should be accomplished in code if possible, but can also be done at the web server level. The header should have a value of `*`, appended only to requests for `/team/*/events.ics`, and optionally any other URIs which could possibly be intended for public, programmatic consumption.

More information about this header: https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS/Errors/CORSMissingAllowOrigin

**Comment by mhall119** [comment]
This header should be on the ical feeds now. Can you check that you're able to do what you wanted with it now?

**Comment by blandin** [comment]
This is odd. In order to make this work on our website, I've been using a reverse proxy in Apache to append the `Access-Control-Allow-Origin` header. This change broke the calendar page because now there are two of those headers, which isn't allowed.

So, I pointed our JavaScript at GT proper because we don't need the reverse proxy anymore, right? Apparently wrong, because then the header does *not* come back on the request.

What's working now is that I'm still using my reverse proxy, but the only line in the vhost config that actually does anything beyond restricting the referer to our website is now commented.

To recap: For some reason, proxied requests for events.ics now have an `Access-Control-Allow-Origin` header, but direct XHR requests do not.

**Comment by mhall119** [comment]
That is quite odd! Could there be a cache somewhere in the middle that's returning the old headers for certain requests?
